distribution/src/assembly/bin.xml

```diff
@@ -63,7 +63,7 @@
 
     <files>
         <file>
-            <source>target/WSO2Carbon/conf/default/deployment.yaml</source>
+            <source>carbon-home/conf/deployment.yaml</source>

```

Comment:
This is the source directory. Default deployment.yaml file is in carbon-home/conf/deployment.yaml. when creating the distribution, we are copying this file to default runtime directory(conf/default/).